2021/omi/OMI-2021-Fichas/settings.json

```
@@ -0,0 +1,29 @@
+{
+  "title": "Juego de Fichas",
+  "source": "OMI 2021 (Diego Garcia Rodríguez del Campo)",
+
+  "limits": {
+    "TimeLimit": 3000,
+    "MemoryLimit": 134217728,
+    "InputLimit": 67108864,
+    "OutputLimit": 67108864,
+    "ExtraWallTime": 0,
+    "OverallWallTimeLimit": 60000
+  },
+
+  "validator": {
+    "name": "custom",
+    "limits": {
+      "TimeLimit": 5000
+    }
+  },
+
+  "misc": {
+    "alias": "omi-2021-fichas",
+    "visibility": "private",
```

```suggestion
    "visibility": "public",
```
Última cosa que falta es que como ahora los problemas son públicos hay que actualizar los `settings.json`.